src/test/java/GoRestTest.java

```diff
@@ -0,0 +1,77 @@
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class GoRestTest {
+
+    @BeforeClass
+    public void setUp() {
+        RestAssured.baseURI = "https://gorest.co.in/";
+        RestAssured.basePath = "public-api/";
+    }
+
+    @Test
+    public void testGoRest() {
```

Comment:
One method should be responsible for single functionality, so please split tests into ~3 methods with more assertions, (creation with validation, deletion with validation, also negative TCs, such as, posting user with the same email)

Reply:
Addressed in e7a4f19.